packages/instant/src/index.umd.ts

```diff
@@ -89,12 +89,12 @@ export const render = (config: ZeroExInstantConfig, selector: string = DEFAULT_Z
     // If the integrator defined a popstate handler, save it to __zeroExInstantIntegratorsPopStateHandler
     // unless we have already done so on a previous render.
     const anyWindow = window as any;
-    if (window.onpopstate && !anyWindow.__zeroExInstantIntegratorsPopStateHandler) {
-        anyWindow.__zeroExInstantIntegratorsPopStateHandler = window.onpopstate.bind(window);
-    }
-    const integratorsOnPopStateHandler = anyWindow.__zeroExInstantIntegratorsPopStateHandler || util.boundNoop;
+    const popStateExistsAndNotSetPreviously = window.onpopstate && !anyWindow.__zeroExInstantIntegratorsPopStateHandler;
+    anyWindow.__zeroExInstantIntegratorsPopStateHandler = popStateExistsAndNotSetPreviously
```

Comment:
this pattern of setting `__zeroExInstantIntegratorsPopStateHandler` to a noop feels kind of weird to me

would the following work instead?

```
anyWindow.__zeroExInstantIntegratorsPopStateHandler = window.onpopstate && !anyWindow.__zeroExInstantIntegratorsPopStateHandler;

const onPopStateHandler = (e: PopStateEvent) => {
        if (anyWindow.__zeroExInstantIntegratorsPopStateHandler) {
             anyWindow.__zeroExInstantIntegratorsPopStateHandler(e):
        }
       ....
}
```